addons/binding/org.openhab.binding.gardena/src/main/java/org/openhab/binding/gardena/internal/model/Device.java

```diff
@@ -110,6 +111,22 @@ public Ability getAbility(String name) throws GardenaException {
         throw new GardenaException("Ability '" + name + "' not found in device '" + this.name + "'");
     }
 
+    public List<Setting> getSettings() {
+        return settings;
+    }
+
+    /**
+     * Returns the setting with the specified name.
+     */
+    public Setting getSetting(String name) throws GardenaException {
+        for (Setting setting : settings) {
+            if (setting.getName().equals(name)) {
```

Comment:
I miss my IDE, in this view I cannot see how items are actually added to the settings?